script.js

```diff
@@ -0,0 +1,41 @@
+const images = document.querySelectorAll('.slider__line img');
+const sliderLine = document.querySelector(' .slider__line');
+let count = 0;
+let width;
+
+function init() {
+    console.log('resize');
+    width = document.querySelector('.slider__block').offsetWidth;
+    sliderLine.style.width = width * images.length + 'px';
```

Comment:
Я б може використовував літерали. Практичної цінності в цьому особливої немає. Просто це волає: "Дивіться, я знаю літерали". Може краще читається.
```suggestion
    sliderLine.style.width = `${width * images.length}px`;
```